src/events/components/CalendarView/CalendarTile.tsx

```diff
@@ -20,15 +22,24 @@ export const createDayList = (amount: number, start: number): number[] => {
   return l;
 };
 
-export const CalendarEventTile = ({ events, active = true, day }: ITileProps) => {
+export const CalendarEventTile = ({ events, active = true, day, month }: ITileProps) => {
+  const thisDate = new Date();
+  const today = thisDate.getDate();
+  const thisMonth = thisDate.getMonth() + 1;
+  // tslint:disable-next-line: radix

```

Comment:
The Radix is a required parameter of the `parseInt` function, or at least it is according to tslint.
The idea is that `parseInt` can parse other number systems than just base 10. `parseInt('11', 10) would give you `11`, while`parseInt('11', 2) would give you `3` because binary. Tslint is just trying to guard against errors here I think